src/log.h

```diff
@@ -119,9 +121,8 @@ class StreamLogOutput : public ICombinedLogOutput {
 
 	void logRaw(LogLevel lev, const std::string &line)
 	{
-		static const std::string use_logcolor = g_settings->get("log_color");
-
-		bool colored = use_logcolor == "detect" ? is_tty : use_logcolor == "yes";
+		bool colored = Logger::use_logcolor == "auto" ? is_tty :
+			Logger::use_logcolor == "always";
```

Comment:
I'd put explicit parentheses around the equivalence expressions just to make the order of operations explicit

Reply:
ok